plugin/pkg/scheduler/factory/factory.go

```diff
@@ -335,14 +336,18 @@ func (f *ConfigFactory) CreateFromConfig(policy schedulerapi.Policy) (*scheduler
 				return nil, err
 			} else {
 				extenders = append(extenders, extender)
+				// Check if bind is offloaded to the extender.
+				if policy.ExtenderConfigs[ii].BindVerb != "" {
+					binder = extender
+				}
 			}
 		}
 	}
-	return f.CreateFromKeys(predicateKeys, priorityKeys, extenders)
+	return f.CreateFromKeys(predicateKeys, priorityKeys, extenders, binder)
 }
 
 // Creates a scheduler from a set of registered fit predicate keys and priority keys.
-func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys sets.String, extenders []algorithm.SchedulerExtender) (*scheduler.Config, error) {
+func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys sets.String, extenders []algorithm.SchedulerExtender, extBinder scheduler.Binder) (*scheduler.Config, error) {
```

Comment:
If you added `IsBinder` to the SchedulerExtender interface, you wouldn't need to pass `extBinder` to this method. You could go over the array of extenders and  create a new binder (`binder{f.client}`) only when none of the extenders is a binder.